packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -103,6 +103,10 @@ import {
 } from '@backstage/plugin-jenkins';
 import { EntityKafkaContent } from '@backstage/plugin-kafka';
 import { EntityKubernetesContent } from '@backstage/plugin-kubernetes';
+import {
+  isKubernetesClustersAvailable,
```

Comment:
Everywhere else this is singular - wonder if it should be here also ?

"isKubernetesClusterAvailable"?